lib/utils.js

```diff
@@ -107,7 +107,14 @@ function prepareInternalLink(link, page)
 
   // Link path
   var linkPath = linkUrl.match(/\d+\.\s+/ig)
-  linkPath = linkPath.slice(0, linkPath.length-1).map(extractIndexes).join('')
+  if(linkPath) {
+    // what is this supposed to do?!
+    // THROWS "Cannot read property 'slice' of null" for empty url
+    linkPath = linkPath.slice(0, linkPath.length-1)
+    linkPath.map(extractIndexes).join('')
```

Comment:
This has a bug, it must be on the same line to do something, result of the join must be assigned to linkPath.